### PR TITLE
Feature/update aquaculture data model

### PR DESCRIPTION
**Pull request by AHABID**

### CODE REVIEW

**Comment by albertoabellagarcia** [comment]
Mandatory (waste of time as well point 1)
1.- please remove README.md and spec.md they are unnecessary and they will be removed when published
2.- missing notes.yaml (even empty) for customization in every data model. See template  https://github.com/smart-data-models/data-models/blob/master/templates/dataModel_for_submision/notes.yaml

Recommendations with possible discussion
1.- Payloads are quite limited (same as schema.json) in terms of attributes
2.- It would be good to populate the ADOPTERS.yaml files

**Comment by albertoabellagarcia** [comment]
Thanks @AHABID. 
Do you need anything else to address these changes?

File: Feed/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Creative Commons Attribution 4.0 International Public License
```
Comment (albertoabellagarcia) [inline Feed/LICENSE.md]: this file is not necessary.

File: Feed/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Feed
```
Comment (albertoabellagarcia) [inline Feed/README.md]: this file is not necessary.

File: Feed/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Entity: Feed
```
Comment (albertoabellagarcia) [inline Feed/doc/spec.md]: this file is not necessary.

File: Feed/examples/example.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Feed/examples/example.json:1]: One property for the payload....  is not much, isn't it?

File: Feed/examples/example.jsonld [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Feed/examples/example.jsonld:1]: One property for the payload....  is not much, isn't it?

File: Feed/schema.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (albertoabellagarcia) [inline Feed/schema.json:1]: One property for the payload....  is not much, isn't it?

File: Feeder/LICENSE.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+# Creative Commons Attribution 4.0 International Public License
```
Comment (albertoabellagarcia) [inline Feeder/LICENSE.md]: this file is not necesary

File: Feeder/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# Feeder
```
Comment (albertoabellagarcia) [inline Feeder/README.md:1]: this file is not necesary

File: Feeder/doc/spec.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+Entity: Feeder
```
Comment (albertoabellagarcia) [inline Feeder/doc/spec.md]: this file is not necesary

File: Feeder/examples/example.json [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Feeder/examples/example.json:1]: one property is not much, isn't it?

File: Feeder/examples/example.jsonld [hunk 11]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Feeder/examples/example.jsonld:1]: one property...

File: Feeder/schema.json [hunk 12]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (albertoabellagarcia) [inline Feeder/schema.json:1]: one property... not much isn't it?

File: FishContainment/doc/spec.md [hunk 13]
```diff
@@ -1,409 +1,49 @@
-Entity: FishContainment  
-=======================  
-[Open License](https://github.com/smart-data-models//dataModel.Aquaculture/blob/master/FishContainment/LICENSE.md)  
-[document generated automatically](https://docs.google.com/presentation/d/e/2PACX-1vTs-Ng5dIAwkg91oTTUdt8ua7woBXhPnwavZ0FxgR8BsAI_Ek3C5q97Nd94HS8KhP-r_quD4H0fgyt3/pub?start=false&loop=false&delayms=3000#slide=id.gb715ace035_0_60)  
+Entity: FishContainment
```
Comment (albertoabellagarcia) [inline FishContainment/doc/spec.md]: this file is not necessary

File: FishContainment/model.yaml [hunk 14]
```diff
@@ -1,320 +0,0 @@
-FishContainment:
```
Comment (albertoabellagarcia) [inline FishContainment/model.yaml:1]: this file is not necessary

File: FishContainment/notes.yaml [hunk 15]
```diff
@@ -1,5 +0,0 @@
-notesHeader:
```
Comment (albertoabellagarcia) [inline FishContainment/notes.yaml:1]: it is required even if there are no notes for the spec. Like it was.

File: FishContainment/schema.json [hunk 16]
```diff
@@ -2,7 +2,7 @@
     "$schema": "http://json-schema.org/schema#",
```
Comment (albertoabellagarcia) [inline FishContainment/schema.json:2]: The definitions of the spec.md have to be included in this file. Please see contribution manual pages 8 and 9.
[https://bit.ly/contribution_manual](https://bit.ly/contribution_manual)

File: FishContainment/examples/example.jsonld [hunk 17]
```diff
@@ -1,17 +1,347 @@
...
+      "unitCode": "MTR"
+    }
+  },
+  "temperature": [
+    {
+      "type": "Property",
+      "value": 15.2,
+      "unitCode": "CEL",
+      "observedAt": "2020-06-26T21:32:52Z",
+      "depth": {
```
Comment (albertoabellagarcia) [inline FishContainment/examples/example.jsonld:50]: I guess this depth does not meet the schema

File: FishContainment/examples/example.jsonld [hunk 18]
```diff
@@ -1,17 +1,347 @@
...
+      "datasetId": "urn:ngsi-ld:Dataset:temperature:Device:02"
+    }
+  ],
+  "dissolvedOxygen": [
+    {
+      "type": "Property",
+      "value": 80,
+      "unitCode": "P1",
+      "observedAt": "2020-06-26T21:32:52Z",
+      "depth": {
```
Comment (albertoabellagarcia) [inline FishContainment/examples/example.jsonld:84]: This does not meet the schema, I guess

File: FishContainment/swagger.yaml [hunk 19]
```diff
@@ -1,52 +0,0 @@
----
```
Comment (albertoabellagarcia) [inline FishContainment/swagger.yaml:1]: this file is not necessary

File: FishPopulation/ADOPTERS.yaml [hunk 20]
```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model FishContainment of the Subject dataModel.Aquaculture.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline FishPopulation/ADOPTERS.yaml:1]: it would be interesting that the use case would be mentioned here.

File: Feed/ADOPTERS.yaml [hunk 21]
```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model FishContainment of the Subject dataModel.Aquaculture.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline Feed/ADOPTERS.yaml:1]: it would be interesting that the use case would be mentioned here.

File: FishPopulation/LICENSE.md [hunk 22]
```diff
@@ -0,0 +1,96 @@
+# Creative Commons Attribution 4.0 International Public License
```
Comment (albertoabellagarcia) [inline FishPopulation/LICENSE.md]: not necesary

File: FishPopulation/README.md [hunk 23]
```diff
@@ -0,0 +1,40 @@
+# FishPopulation
```
Comment (albertoabellagarcia) [inline FishPopulation/README.md]: not necessary

File: FishPopulation/doc/spec.md [hunk 24]
```diff
@@ -0,0 +1,25 @@
+Entity: FishPopulation
```
Comment (albertoabellagarcia) [inline FishPopulation/doc/spec.md]: not necessary

File: FishPopulation/examples/example.jsonld [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:FishContainment:01",
+    "observedAt": ""
+  },
+  "refSpecie":{
+    "type": "Relationship",
+    "object":"urn:ngsi-ld:Specie:01"
+  },
+  "@context": [
+    "https://raw.githubusercontent.com/smart-data-models/data-models/master/context.jsonld"
```
Comment (albertoabellagarcia) [inline FishPopulation/examples/example.jsonld:27]: Better the local context of the subject

File: FishPopulation/schema.json [hunk 26]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/dataModel.Agrifood/agrifood-schema.json#/definitions/AgriFood-Commons"
+        },
+        {
+            "properties": {
+                "type": {
```
Comment (albertoabellagarcia) [inline FishPopulation/schema.json:21]: shouldn't be FishPopulation?

File: Specie/ADOPTERS.yaml [hunk 27]
```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model FishContainment of the Subject dataModel.Aquaculture.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline Specie/ADOPTERS.yaml:1]: It would be good to mention the use case here

File: Specie/LICENSE.md [hunk 28]
```diff
@@ -0,0 +1,96 @@
+# Creative Commons Attribution 4.0 International Public License
```
Comment (albertoabellagarcia) [inline Specie/LICENSE.md]: not necessary

File: Specie/doc/spec.md [hunk 29]
```diff
@@ -0,0 +1,25 @@
+Entity: Specie
+=======================
```
Comment (albertoabellagarcia) [inline Specie/doc/spec.md]: Definitions here should be included in the schema. See contribution manual pages 8 and 9.
[https://bt.ly/contribution_manual](https://bt.ly/contribution_manual)

File: Specie/examples/example.json [hunk 30]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Specie/examples/example.json:1]: one attribute...

File: Specie/examples/example.jsonld [hunk 31]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (albertoabellagarcia) [inline Specie/examples/example.jsonld]: one attribute...

File: Specie/schema.json [hunk 32]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/dataModel.Agrifood/agrifood-schema.json#/definitions/AgriFood-Commons"
+        },
+        {
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "FishContainment"
```
Comment (albertoabellagarcia) [inline Specie/schema.json:24]: it should be Spiece isn't it?

File: Sump/ADOPTERS.yaml [hunk 33]
```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model FishContainment of the Subject dataModel.Aquaculture.  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline Sump/ADOPTERS.yaml:1]: It can be included the use case here

File: Sump/LICENSE.md [hunk 34]
```diff
@@ -0,0 +1,96 @@
+# Creative Commons Attribution 4.0 International Public License
```
Comment (albertoabellagarcia) [inline Sump/LICENSE.md]: not necessary

File: Sump/README.md [hunk 35]
```diff
@@ -0,0 +1,40 @@
+# Sump
```
Comment (albertoabellagarcia) [inline Sump/README.md]: not necessary

File: Sump/doc/spec.md [hunk 36]
```diff
@@ -0,0 +1,31 @@
+Entity: Sump
```
Comment (albertoabellagarcia) [inline Sump/doc/spec.md]: definitions here to the shcema.json according to the [contribution manual](https://bit.ly/contribution_manual) pages 8 and 9

File: Sump/schema.json [hunk 37]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (albertoabellagarcia) [inline Sump/schema.json:1]: missing attributes used in the payloads.

**Review by albertoabellagarcia — COMMENTED** [review]
We need a call for saving your time. https://doodle.com/meeting/participate/id/QeZ0Mlgb

**Review by albertoabellagarcia — COMMENTED** [review]
please address the comments

File: context.jsonld [hunk 38]
```diff
@@ -1,24 +1,9 @@
 {
```
Comment (albertoabellagarcia) [inline context.jsonld]: This file is not necessary is also generated automatically

**Review by albertoabellagarcia — APPROVED** [review]

**Review by albertoabellagarcia — APPROVED** [review]

**Comment by AHABID** [comment]
I will merge this PR since it was opened since long time, and create new one about:
1. attributes definitions and possible alignements
2. digital twin integration

**Pull request merged**